src/environments/environment.prod.ts

```diff
@@ -8,12 +8,9 @@ import { SHARED } from './shared';
 export const environment = {
   ...SHARED,
 
-  CLIENTE_PRESUPUESTO: 'https://presupuesto.portaloas.udistrital.edu.co/',
-  CLIENTE_CONTABILIDAD: 'https://contabilidad.portaloas.udistrital.edu.co/',
   production: true,
```

Comment:
```suggestion
  CLIENTE_PRESUPUESTO: 'https://presupuesto.portaloas.udistrital.edu.co/',
  CLIENTE_CONTABILIDAD: 'https://contabilidad.portaloas.udistrital.edu.co/',
  production: true,
```

De paso, abajo en el `TOKEN`, actualizar las siguientes propiedades:
```typescript
{
REDIRECT_URL: 'https://planadquisiciones.portaloas.udistrital.edu.co/',
SIGN_OUT_REDIRECT_URL: 'https://planadquisiciones.portaloas.udistrital.edu.co/',
}
```